python-template/{{cookiecutter.placeholder_repo_name}}/openedx.yaml

```diff
@@ -21,3 +21,9 @@ oeps:
         reason: TODO - Implement for this application if appropriate
     oep-30:
         state: True
+openedx-release:
+  # The openedx-release key is described in OEP-10:
```

Comment:
You need an .editorconfig file in this repo.  This file now has mixed 2- and 4-space indents.